.github/ISSUE_TEMPLATE/1. yaml-error-or-warning.md

```diff
@@ -20,7 +20,7 @@ labels: 'yaml'
 2.  Is a HA scheme error?
 
     Feel free to fix this yourself, it's quite easy, HowTo here:
-    https://github.com/keesschollaart81/vscode-home-assistant/wiki/HowTo:-Update-the-schemas
+    https://github.com/keesschollaart81/vscode-home-assistant/wiki/HowTo:-Update-the-schema's
```

Comment:
The comma should not be there... that is a GitHub bug I would say. For now wrapping it, as auto-linking doesn't pick it up correctly either 😞 .

```suggestion
    <https://github.com/keesschollaart81/vscode-home-assistant/wiki/HowTo:-Update-the-schema's>
```